api/base/authentication/drf.py

```diff
@@ -50,8 +50,9 @@ def authenticate_credentials(self, userid, password):
             raise exceptions.NotAuthenticated()
         return (user, None)
 
+    # Returns custom value other than "Basic" to prevent BasicAuth dialog prompt when returning 401
     def authenticate_header(self, request):
-        return ""
+        return 'Documentation realm="%s"' % self.www_authenticate_realm
```

Comment:
Not a big issue here, but `string.format` is the more modern and preferred way  to insert values (as opposed to `"%s" % value). (see PEP3101)